src/hooks/useLoginHandler.js

```diff
@@ -34,14 +35,19 @@ function useLoginHandler() {
       localStorage.setItem("authUser", user);
       response = await getAllPlaylistsFromServer(tokenResponse);
       dispatch({
-        type: "PLAYLIST_OPERATION",
+        type: "SET_PLAYLISTS",
         payload: { playlists: response.playlists },
       });
       response = await getAllVideosInHistoryFromServer(tokenResponse);
       dispatch({
-        type: "HISTORY_OPERATION",
+        type: "SET_HISTORY",
```

Comment:
You can separate your `type` in a separate file. 

`const ACTION_TYPE = {
  SET_HISTORY: 'SET_HISTORY'
}`

then you can use these action types in your dispatch and reducer, this will be less error-prone and readable also. 

But, it's up to your preference.